### PR TITLE
prepare v1.3.0rc

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

prepare to have RC for future 1.3.0

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -48,6 +48,7 @@
 # Testing
 /tests/helpers/boring_model.py          @williamfalcon @tchaton @borda
 
-/.github/CODEOWNERS     @williamfalcon
-/README.md              @williamfalcon @edenlightning @borda
-/setup.py               @williamfalcon @borda
+/.github/CODEOWNERS         @williamfalcon
```
Comment (tchaton) [inline .github/CODEOWNERS:51]: Can you add Carlos and I for the profilers ?

**Review by tchaton — APPROVED** [review]
LGTM !

File: .github/CODEOWNERS (hunk 1 above)
Comment (tchaton) [reply .github/CODEOWNERS:51]: And @kaushikb11 for TPU Plugins.

Comment (Borda) [reply .github/CODEOWNERS:51]: you are already there for profilers :]
@kaushikb11 mind suggest yourself where to put you

Comment (kaushikb11) [reply .github/CODEOWNERS:51]: And other places as well. Would like a notification to do reviews :)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -27,7 +27,7 @@
 /pytorch_lightning/loggers              @tchaton @awaelchli @borda
 /pytorch_lightning/overrides            @tchaton @SeanNaren @borda
 /pytorch_lightning/plugins              @tchaton @SeanNaren @awaelchli @justusschock
-/pytorch_lightning/profiler             @williamfalcon @tchaton @borda
+/pytorch_lightning/profiler             @williamfalcon @tchaton @borda @carmocca
```
Comment (Borda) [inline .github/CODEOWNERS:30]: @tchaton you are already there for profilers :]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -5,7 +5,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-* @williamfalcon @borda @tchaton @SeanNaren @carmocca @awaelchli @justusschock
+* @williamfalcon @borda @tchaton @SeanNaren @carmocca @awaelchli @justusschock @kaushikb11
```
Comment (Borda) [inline .github/CODEOWNERS:8]: @kaushikb11 this brings you notifications ;]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Borda) [reply .github/CODEOWNERS:51]: @kaushikb11 for general notifications https://github.com/PyTorchLightning/pytorch-lightning/pull/6885/files#r609448605
no pls suggest the specific as code-owner

Comment (kaushikb11) [reply .github/CODEOWNERS:51]: I have pushed the suggestions!

**Review by kaushikb11 — APPROVED** [review]

**Review by SeanNaren — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by Borda** [comment]
@kaushikb11 ok, the problem was with using py3.6

File: .circleci/config.yml [hunk 4]
```diff
@@ -36,11 +36,11 @@ references:
   build_push_docker: &build_push_docker
     run:
       name: Build and push Docker image
+      environment:
+        - PYTHON_VER: 3.6
```
Comment (Borda) [inline .circleci/config.yml:40]:
```suggestion
        - PYTHON_VER: 3.7
```

**Review by ananthsub — APPROVED** [review]

**Pull request merged**